### PR TITLE
New version: Microsoft.PowerShell version 7.4.0

### PR DESCRIPTION
**Pull request by pwshBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127003)

### CODE REVIEW

File: manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.0
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.installer.yaml:5]:
```suggestion
PackageVersion: 7.4.0.0
```
PackageVersion should match the version written to Control Panel / Registry. This is to help WinGet do correct package correlation / upgrade matching

File: manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.0
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.4.0.0
```
PackageVersion should match the version written to Control Panel / Registry. This is to help WinGet do correct package correlation / upgrade matching

File: manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.0
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.yaml:5]:
```suggestion
PackageVersion: 7.4.0.0
```

PackageVersion should match the version written to Control Panel / Registry. This is to help WinGet do correct package correlation / upgrade matching

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, the version folder would also require renaming to match the new PackageVersion. The folder under `manifests/m/Microsoft/PowerShell` should be renamed from `7.4.0` -> `7.4.0.0`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by mdanish-kh** [comment]
cc @anamnavi

**Comment by mdanish-kh** [comment]
[Policy] Needs-Author-Feedback

https://github.com/microsoft/winget-pkgs/pull/127003#pullrequestreview-1735247391

**Comment by anamnavi** [comment]
@mdanish-kh thanks for catching that and pointing out that that would require the PackageVersion to match. Due to access to the pwshbot account, I can't fix this directly here and will fix it in our automation for next time. I've created a PR #127133  with the YamlCreate script and correct version and will close this one in favor of that.

**Comment by denelon** [comment]
Closing in favor of :
* https://github.com/microsoft/winget-pkgs/pull/127133

**Pull request closed without merging**